### DISCUSSION TITLE
What does hitgroup 8 mean?

### DISCUSSION THREAD

**Discussion by Dying-Ducks** [akiver/cs-demo-manager#824] [Q&A] [answered]
In the "Weapons" tab one can see a hit group graph, which shows where your hits went.

I noticed that sometimes the table left doesn’t match the graph right, like in this example:
![Screenshot 2024-05-05 161229](https://github.com/akiver/cs-demo-manager/assets/109040754/d4a2c8b7-1f4c-444b-8885-97ba41052a7e)

Where 200 Damage were made, but only 173 where displayed in the hitgroup graph. So we miss a hit for 27 damage. 

After a little search in the .json of that demo I could find, that hitgroup 1 refers to the head, 2 to the Chest and so on until 7 refers to the right leg.
And I figured out that the missing hit was on the "hitgroup 8".  So every hit in that group 8 will show up on the left side, but not on the right. 

I wonder what "hitgroup 8" even means and why it don't show up in the right grahic that results in not matching data.
I checked the demo on that example and it was a hit to the upper body.

**Comment by Dying-Ducks** [comment]
And when we at the topic codes:
Whats up with end_reason?
Bomb explosion is 1, bomb defusion is 7. Whats with 2-6?
Elimination CT is 8 and Elimination T is 9, but time running out is 12. 
I never saw any round with 10 or 11.

**Comment by akiver** [comment, marked as answer]
Hitgroup 8 is the neck, it will be added in the next version.
About round end reasons you can find all known values in this file https://github.com/akiver/cs-demo-analyzer/blob/main/js/src/constants.ts
